km/km.h

```diff
@@ -362,6 +362,9 @@ typedef struct km_machine {
 
    uint64_t xcr0;
    int overcommit_memory;   // controls how we request memory for payload from Linux
+
+   km_gva_t snap_rcvr_brk;
```

Comment:
Do we really need to keep these in here - looks like we only need them for a very short time recovering from the snap?